modules/release-notes/pages/index.adoc

```diff
@@ -19,9 +19,9 @@ TigerGraph Server 3.9.2 LTS was released on June 29, 2023.
 
 TigerGraph Server 3.9.1 LTS was released on March 22, 2023.
 
-TigerGraph Server 3.9.0 LTS was released on February 28th, 2023.
+TigerGraph Server 3.9.0 Preview was released on February 28th, 2023.
```

Comment:
Please make some additional changes to this list of dates:
TigerGraph Server 3.9.3-3 LTS was released on February 15th, 2024.
TigerGraph Server 3.9.3-2 LTS was released on January 16th, 2024.
TigerGraph Server 3.9.3-1 LTS was released on December 5th, 2023.
etc.
For dates, even though we **say** "fifteenth" not "fifteen", it is traditional not to **write** the "th". (or "st", "nd", "rd") in dates.
So, it should be
TigerGraph Server 3.9.3-3 LTS was released on February 15, 2024.
TigerGraph Server 3.9.3-2 LTS was released on January 16, 2024.
TigerGraph Server 3.9.3-1 LTS was released on December 5, 2023.
etc.